### PR TITLE
[15.0][MIG]account_fiscal_position_partner_type

### PR DESCRIPTION
**Pull request by manuelregidor**
Standard migration.

### CODE REVIEW

**Comment by manuelregidor** [comment]
@ValentinVinagre @HaraldPanten

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍  @ValentinVinagre

File: account_fiscal_position_partner_type/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Account Fiscal Position Partner Type",
-    "version": "14.0.1.0.0",
+    "version": "15.0.1.0.0",
```
Comment (ValentinVinagre) [inline account_fiscal_position_partner_type/__manifest__.py:6]: The version number should not be changed in the imp, it should be changed in the migration commit

**Comment by manuelregidor** [comment]
@sbidoul I'm getting the following error. Do you know why it might be failing? Thank you.

ERROR: Could not find a version that satisfies the requirement setuptools-odoo>=2.7 (from versions: none)
ERROR: No matching distribution found for setuptools-odoo>=2.7

**Comment by dreispt** [comment]
I believe the [OCA Wheelhouse](https://wheelhouse.odoo-community.org) certificate has expired.

**Comment by sbidoul** [comment]
Certificate renewal on odoo-community.org was failing Not sure why yet but they are now refreshed.

**Comment by sbidoul** [comment]
I renewed the certificates.

**Comment by pedrobaeza** [comment]
Thanks!

**Comment by HaraldPanten** [comment]
@manuelregidor can you rebase?

THX

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Comment by HaraldPanten** [comment]
@pedrobaeza @dreispt is it OK for you if we merge this one for not blocking other dependencies?

THX

**Pull request merged**